Reference/Searching/Examine/Indexing/index.md

```diff
@@ -22,212 +13,123 @@ You can modify the built in indexes in several ways:
+:::
 
-Starting in Umbraco 8.4.0, the `IValueSetValidator` implementation for the built in indexes, can be changed like this:
+When using the `ConfigureNamedOptions` pattern, we have to register this in a composer for it to configure our indexes, this can be done like this:
 
 ```c#
-using Examine;
-using Umbraco.Core;
-using Umbraco.Core.Composing;
-using Umbraco.Core.Services;
-using Umbraco.Examine;
-
-public class CustomizeIndexComposer : IUserComposer
-{
-    public void Compose(Composition composition)
-    {
-        // replace the default IUmbracoIndexConfig definition
-        composition.RegisterUnique<IUmbracoIndexConfig, CustomIndexConfig>();
-    }
-}
+using Microsoft.Extensions.DependencyInjection;
+using Umbraco.Cms.Core.Composing;
+using Umbraco.Cms.Core.DependencyInjection;
 
-// inherit from the default
-public class CustomIndexConfig : UmbracoIndexConfig, IUmbracoIndexConfig
+namespace MySite
 {
-    public CustomIndexConfig(IPublicAccessService publicAccessService) : base(publicAccessService)
+    public class ExamineComposer : IComposer
     {
-    }
-
-    // explicit implementation - overrides base class.
-    // GetMemberValueSetValidator is used for the MemberIndex.
-    IValueSetValidator IUmbracoIndexConfig.GetMemberValueSetValidator()
-    {
-        // NOTE - by default Umbraco only includes a small subset of fields for the member index for
-        // security reasons, however you may want to include a larger set of data in the member index.
-        // This example will include all data for a member excluding the built in property values.
-
-        // all built in member properties to exclude (i.e. Password question, etc...)
-        var excludeFields = Constants.Conventions.Member.GetStandardPropertyTypeStubs().Keys;
-
-        // include everything except the above
-        return new MemberValueSetValidator(null, null, null, excludeFields);
+        public void Compose(IUmbracoBuilder builder)
+        {
+            builder.Services.ConfigureOptions<ConfigureExamineOptions>();
+        }
     }
 }
 ```
+### Changing field value types
 
-If you are using a version of Umbraco before 8.4.0 then to do this you will need to modify the index creation (see below).
-
-### Overriding index creation
-
-You can completely take control of the whole Umbraco index creation logic by replacing the default implementation of `IUmbracoIndexesCreator`.
-
-A few examples of why you might want to override the index creation:
+By default Examine will store values into the Lucene index as "Full Text", meaning it will be indexed and analyzed for a textual search. However, if a value that you are storing in a field is numerical, a date/time or another specific value type then you might want to change how this is stored in the index. This way you will be able to take advantage of some features such as numerical range or date features, etc...
```

Comment:
```suggestion
By default Examine will store values into the Lucene index as "Full Text" fields, meaning the values will be indexed and analyzed for a textual search. However, if a field value is numerical, date/time or another non-textual value type, you might want to change how the value is stored in the index. This will let you take advantage of some value type specific search features such as numerical or date range.
```